client/app.py

```diff
@@ -11,59 +11,61 @@ def get_user_select(data):
     select = data[int(input())]
     return select
 
+
 def method_and_status(data):
-    keys= {}
-    for i, key in enumerate (data) :
+    keys = {}
+    keys[10] = 'exit'
+    for i, key in enumerate(data):
         print(f"{i}. {key} status: {data[key]}")
-        keys = { i : keys}
-    select = data[keys[int(input())]]
-    return select    
+        keys[i] = key
+    print('10 jeśli skończyłeś')
+    select_method = keys[int(input())]
+    return select_method
+
 
 def new_device():

```

Comment:
What is the difference between `new_device` and `add_device`? The naming should be more adequate.